sdk/include/ddk/ntstrsafe.h

```diff
@@ -2825,6 +2826,45 @@ RtlStringLengthWorkerW(
     return Status;
 }
 
+NTSTRSAFEAPI 
+RtlUnicodeStringPrintf(PUNICODE_STRING DestinationString,
+	NTSTRSAFE_PCWSTR pszFormat, ...)
+{
+	if (DestinationString == NULL || pszFormat == NULL)
+		return STATUS_INVALID_PARAMETER;
+
+	if (DestinationString->Buffer == NULL || DestinationString->MaximumLength == 0)
+		return STATUS_INVALID_PARAMETER;
+	
+	size_t cchLength = 0;
+	NTSTATUS nt = RtlStringLengthWorkerW(DestinationString->Length, NTSTRSAFE_UNICODE_STRING_MAX_CCH, &cchLength);
```

Comment:
Same comment for this one. Also, status variables are called "Status", not "nt".